lib/cisco_node_utils/acl.rb

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # Acl - node utility class for ACL configuration
+  class Acl < NodeUtil
+    attr_reader :acl_name, :afi
+
+    # acl_name: name of the acl
+    # instantiate: true = create acl instance
+    def initialize(acl_name, afi, instantiate=true)
```

Comment:
nit: This is a bit of an afterthought but the order of your parameters may make better sense if swapped; ie. should your manifest titles be:
 ` cisco_acl { 'acl_name  ipv4':`  or would they be better as `cisco_acl { 'ipv4 acl_name': ` ? The difference is minor but I would probably order my manifest by afi's, so all ipv4 acls and then all the ipv6 acls. But like I said this is just a nit so you can keep it as is if you prefer.